### PR TITLE
Containerization

### PR DESCRIPTION
**Pull request by ivan-c**
- Add Dockerfile for building production version of VueJS app
- Add labels and networking needed to deploy with traefik (move to separate YAML override?)
- Remove hardcoded reference to GitHub Pages deploy (upstream)

### CODE REVIEW

**Review by mcjustin — APPROVED** [review]

**Review by jhowe-uw — APPROVED** [review]

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+services:
+
+  # expose client to internet
+  client:
+    build: .
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.client-${COMPOSE_PROJECT_NAME}.rule=Host(`client.${BASE_DOMAIN:-asbi-test.cirg.uw.edu}`)"
```
Comment (ivan-c) [inline docker-compose.yaml:9]: This is pretty much the URL pattern (`$DC_SERVICE.$BASE_DOMAIN`) I use by default everywhere. If this pattern were the default router(?) in our infra traefik config, I could remove this line from every YAML file.

@jhowe-uw is that something you would be able to do in the infra traefik config? LMK if it'd be helpful to have a Cosri PivotalTracker story

**Review by pbugni — APPROVED** [review]
Nice.

File: docker-compose.yaml (hunk 1 above)
Comment (ivan-c) [reply docker-compose.yaml:9]: This is now captured in the below PT story:
[PT#181796672](https://www.pivotaltracker.com/story/show/181796672)

**Comment by ivan-c** [comment]
@achen2401 When you get a chance, could you review this (if you haven't already)? 

In the long run, I'd like to work with you so I get better at using docker with frontend projects, and walk you through how to dockerize a frontend project from scratch

**Review by achen2401 — APPROVED** [review]

**Pull request merged**